homatransport/src/application/WorkloadSynthesizer.cc

```diff
@@ -143,6 +146,11 @@ WorkloadSynthesizer::initialize()
     startTime = par("startTime").doubleValue();
     stopTime = par("stopTime").doubleValue();
     xmlConfig = par("appConfig").xmlValue();
+    std::string OutputFileName = std::string(
+                "results/") + std::string(par("resultFile").stringValue());
+    if(!outputFile.is_open()) {
```

Comment:
Why is this needed? Doesn't the omnet++ infrastructure open/close the result file itself?

Reply:
Using my own logfiles over the default logging in omnet right now.